tests/includes/ApplicationBaseTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Matthew
+ * Date: 2/22/16
+ * Time: 13:53
+ */
+
+namespace Waca\Tests;
+
+class ApplicationBaseTest extends \PHPUnit_Framework_TestCase
+{
+	private $si;
+    private $ab;
+
+	public function setUp() {
+		$this->si = new \Waca\siteConfiguration();
+
+		$this->ab = $this->getMockForAbstractClass('\Waca\ApplicationBase', [$this->si]);
```

Comment:
While `'\Waca\ApplicationBase'` works, but risks not being picked up by code analysis tools which don't understand this is a class reference.

[Since PHP 5.5](https://secure.php.net/manual/en/language.oop5.basic.php#language.oop5.basic.class.class), we can use `::class` to get this automatically handled for us